src/features/components/templates/footer/disclaimer/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react'
+import { risk_warning_container } from './disclaimer.module.scss'
+import { disclaimerContent, riskWarningContent } from './data'
+import { DisclaimerParagraphConfig } from './types'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import useVisibleContent from 'components/hooks/use-visible-content'
+import useRegion from 'components/hooks/use-region'
+import Container from 'features/components/atoms/container'
+
+const Disclaimer = () => {
+    const { is_eu, is_non_eu, is_cpa_plan } = useRegion()
+
+    const visibility_config: DisclaimerParagraphConfig = useMemo(() => {
+        return {
+            is_cpa_plan,
+            is_eu,
+            is_non_eu,
+            is_non_eu_or_cpa_plan: is_non_eu || is_cpa_plan,
+        }
+    }, [is_cpa_plan, is_eu, is_non_eu])
```

Comment:
In case when we'll use this approach with `visibility` property in data, can we import it from context via `useRegion`?
Or we have some limitations in context?